client/knowledgebase/components/Category.jsx.orig

```diff
@@ -0,0 +1,75 @@
+import React, { PropTypes } from 'react';
```

Comment:
Delete unused files like this. We have git history.

Reply:
done. also added to it .gitignore